modules/stanford_earth_capx/src/EarthCapxInfo.php

```diff
@@ -0,0 +1,249 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: kennethsharp1
+ * Date: 5/30/18
+ * Time: 9:29 PM
+ */
+
+namespace Drupal\stanford_earth_capx;
+
+use Drupal\Core\Database;
+use Drupal\migrate\MigrateMessage;
+
+/*
+ * Encapsulates an information table for CAP-X Profile imports
+ */
+class EarthCapxInfo
+{
+
+  const EARTH_CAPX_INFO_INVALID = 0;
+  const EARTH_CAPX_INFO_NEW = 1;
+  const EARTH_CAPX_INFO_FOUND = 2;
+
+  const EARTH_CAPX_INFO_TABLE = 'migrate_info_earth_capx_importer';
+
+  private $sunetid;
```

Comment:
Missing comments.

Reply:
Comment added.

Reply:
I'm not seeing comments but I won't let that stop this pr.